contrib/vspec2graphql/README.md

```diff
@@ -0,0 +1,57 @@
+## What does vspec2graphql do?
+
+This allows to automatically generate a graphql schema that can represent VSS data.
+The resulting schema does only allow querying information. Mutations are not supported.
+
+The resulting schema will look something like this:
+```
+type Query {
+  vehicle(
+    """VIN of the vehicle that you want to request data for."""
+    id: String!
+
+    """
+    Filter data to only provide information that was sent from the vehicle after that timestamp.
+    """
+    after: String
+  ): Vehicle
+}
+
+"""Highlevel vehicle data."""
+type Vehicle {
+  """Attributes that identify a vehicle"""
+  vehicleIdentification: Vehicle_VehicleIdentification
+  ...
+}
+
+type Vehicle_VehicleIdentification {
+  ...
+}
+...
+```
+
+Leaves look like this:
+```
+"""Vehicle brand or manufacturer"""
+type Vehicle_VehicleIdentification_Brand {
+  """Value: Vehicle brand or manufacturer"""
+  value: String
+
+  """Source system: Vehicle brand or manufacturer"""
+  source: String
```

Comment:
A few lines describing the intended usage of `source` and `channel` would be helpful.

Reply:
Default scalar types in GraphQL:

    Int: A signed 32‐bit integer.
    Float: A signed double-precision floating-point value.
    String: A UTF‐8 character sequence.
    Boolean: true or false.
    ID: The ID scalar type represents a unique identifier, often used to refetch an object or as the key for a cache.

Reply:
My original thought was to just have a few lines describing if the contents/use of the fields are implementation specific or preferably shall be standardized.  Similar for timestamp - shall a specific format (like ISO 8601) be used or is it that implementation specific.

Maybe just add something similar to below to the readme:


_`source` and `channel` are free text fields which can be used to include information on the origin of the data. Content is implementation specific. `timestamp` shall be the timestamp of the observation formatted according to ISO 8601 with UTC time zone._

Reply:
Thanks